config.go

```diff
@@ -56,6 +62,10 @@ var defaultConfig = Config{
 		CountThresh:       5,
 		NonzeroMaxPercent: 50,
 	},
+	LEDs: LEDsConfig{
+		Recording: "GPIO20",
+		Power:     "GPIO21",
```

Comment:
Should this be called "running" or "active" rather than "power"? It's more that the recorder is active than the device is powered on.